### ISSUE TITLE
Server Crash with SeasonAPI 1.1.3b

### ISSUE THREAD

**Issue by derSchotte** [EdwinMindcraft/SeasonsAPI#5]
Hi,

After two hours of gameplay und chancing from autonum to winter the server crashes...

`---- Minecraft Crash Report ----

WARNING: coremods are present:
  LoadingPlugin (Quark-beta-18.jar)
  NowWithRendering (redstonepaste-mc1.9-1.7.3.jar)
  FldLoadingPlugin (FastLeafDecay-1.9-1.6.jar)
  LoadingPlugin (RandomThings-MC1.9-3.7.jar)
  LoadingPluginBuildingBricks (BuildingBricks-1.9-2.0.1.jar)
  SeasonsPreLoader (SeasonsAPI-1.1.3b.jar)
  MMFMLCorePlugin (MultiMine-1.9.jar)
  MalisisCorePlugin (malisiscore-1.9-3.0.6.jar)
  MalisisSwitchesPlugin (malisisswitches-1.9-2.0.0.jar)
  CoreMod (Aroma1997Core-1.9-1.0.5.0.b65.jar)
  Reborn Core ASM (RebornCore-1.9-2.3.2.84-universal.jar)
Contact their authors BEFORE contacting forge

// You should try our sister game, Minceraft!

Time: 23.05.16 18:39
Description: Exception in server tick loop

java.lang.NoSuchMethodError: net.minecraft.world.World.func_72894_k(F)V
    at mod.mindcraft.seasons.WorldHandler.worldTick(WorldHandler.java:178)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_322_WorldHandler_worldTick_WorldTickEvent.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:49)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:140)
    at net.minecraftforge.fml.common.FMLCommonHandler.onPreWorldTick(FMLCommonHandler.java:274)
    at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:690)
    at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:386)
    at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:609)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:467)
    at java.lang.Thread.run(Unknown Source)
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- System Details --
Details:
    Minecraft Version: 1.9
    Operating System: Windows Server 2008 R2 (amd64) version 6.1
    Java Version: 1.8.0_91, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 2654284144 bytes (2531 MB) / 4127719424 bytes (3936 MB) up to 4127719424 bytes (3936 MB)
    JVM Flags: 2 total; -Xmx4096M -Xms4096M
    IntCache: cache: 16, tcache: 0, allocated: 4, tallocated: 105
    FML: MCP 9.23 Powered by Forge 12.16.1.1900 151 mods loaded, 151 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCHIJAAAA   mcp{9.19} [Minecraft Coder Pack](minecraft.jar) 
    UCHIJAAAA   FML{8.0.99.99} [Forge Mod Loader](minecraft_server.jar) 
    UCHIJAAAA   Forge{12.16.1.1900} [Minecraft Forge](minecraft_server.jar) 
    UCHIJAAAA   Aroma1997Core{${version}} [Aroma1997Core](Aroma1997Core-1.9-1.0.5.0.b65.jar) 
    UCHIJAAAA   buildingbricks{1.9-2.0.1} [Building Bricks](BuildingBricks-1.9-2.0.1.jar) 
    UCHIJAAAA   JEI{3.3.3.200} [Just Enough Items](jei_1.9-3.3.3.200.jar) 
    UCHIJAAAA   abyssalcraft{1.9.1.6} [AbyssalCraft](AbyssalCraft-1.9-1.9.1.6.jar) 
    UCHIJAAAA   mantle{1.9-0.9.4.jenkins114} [Mantle](Mantle-1.9-0.9.4.jar) 
    UCHIJAAAA   tconstruct{1.9-2.2.2.jenkins156} [Tinkers' Construct](TConstruct-1.9-2.2.2.jar) 
    UCHIJAAAA   acintegration{1.4.2.2} [AbyssalCraft Integration](AbyssalCraft Integration-1.9-1.4.2.2.jar) 
    UCHIJAAAA   actuallyadditions{1.9-r33} [Actually Additions](ActuallyAdditions-1.9-r33.jar) 
    UCHIJAAAA   additionalbanners{1.1.0.11} [Additional Banners](AdditionalBanners-1.9-1.1.0.13.jar) 
    UCHIJAAAA   appleagain{1.1} [Apples Again](ApplesAgain-1.9.0-1.1.jar) 
    UCHIJAAAA   armorplus{1.11.0-1.9} [ArmorPlus](ArmorPlus1.11.0%281.9%29.jar) 
    UCHIJAAAA   Aroma1997CoreHelper{1.0.5.0.b65} [Aroma1997Core|Helper](Aroma1997Core-1.9-1.0.5.0.b65.jar) 
    UCHIJAAAA   AromaBackup{1.0.1.0.b31} [AromaBackup](AromaBackup-1.9-1.0.1.0.b31.jar) 
    UCHIJAAAA   AromaBackupRecovery{1.0} [AromaBackup Recovery](AromaBackup-1.9-1.0.1.0.b31.jar) 
    UCHIJAAAA   bagginses{2.3.0b} [Bagginses](Bagginses-1.9-2.3.0b.jar) 
    UCHIJAAAA   Baubles{1.2.0.2} [Baubles](Baubles-1.9-1.2.0.2.jar) 
    UCHIJAAAA   baublesstuff{2.0.8} [Baubles Stuff](baublesstuff-2.0.8.jar) 
    UCHIJAAAA   bdlib{1.11.0.14} [BD Lib](bdlib-1.11.0.14-mc1.9.jar) 
    UCHIJAAAA   BetterAchievements{0.2.5.19} [Better Achievements](BetterAchievements-1.9-0.2.5.19.jar) 
    UCHIJAAAA   BBG{4.0.0} [BetterBedrockGen](BetterBedrockGenerator-1.9.0-3.0.0.jar) 
    UCHIJAAAA   betterbuilderswands{0.6.1} [Better Builder's Wands](BetterBuildersWands-0.6.1-1.9r128+baa0aff.jar) 
    UCHIJAAAA   BiomesOPlenty{4.0.2.2018} [Biomes O' Plenty](BiomesOPlenty-1.9-4.0.2.2018-universal.jar) 
    UCHIJAAAA   BloodMagic{1.9-2.0.0-38} [Blood Magic: Alchemical Wizardry](BloodMagic-1.9-2.0.0-38.jar) 
    UCHIJAAAA   bookshelf{1.2.0.237} [Bookshelf](Bookshelf-1.9-1.2.0.241.jar) 
    UCHIJAAAA   bookshelfapi{3.0.4} [Bookshelf](Bookshelf-API-1.9-3.0.4.jar) 
    UCHIJAAAA   Botania{r1.8-304} [Botania](Botania-unofficial.r1.8-304.jar) 
    UCHIJAAAA   Chameleon{1.9-2.1.3} [Chameleon](Chameleon-1.9-2.1.3.jar) 
    UCHIJAAAA   chisel{@VERSION@} [Chisel](Chisel-MC1.9-0.0.4.14.jar) 
    UCHIJAAAA   mcmultipart{1.1.1} [MCMultiPart](MCMultiPart-1.1.1-universal.jar) 
    UCHIJAAAA   chiselsandbits{9.18} [Chisels & Bits](chiselsandbits-9.18.jar) 
    UCHIJAAAA   claybucket{1.0} [Clay Bucket](ClayBucket-1.9-1.0.jar) 
    UCHIJAAAA   cyclopscore{0.5.9} [Cyclops Core](CyclopsCore-1.9-0.5.9.jar) 
    UCHIJAAAA   colossalchests{1.3.4} [Colossal Chests](ColossalChests-1.9-1.3.4.jar) 
    UCHIJAAAA   compactstorage{v2.0.0a} [CompactStorage](compactstorage-2.1-1.9-rc.jar) 
    UCHIJAAAA   compasscoords{1.9-3.0.2} [Compass Co-ords](CompassCoords-1.9-3.0.2.jar) 
    UCHIJAAAA   craftablenametags{1.2} [Craftable Name Tags](craftablenametags-1.2.jar) 
    UCHIJAAAA   craftingcraft{3.0.3} [CraftingCraft](craftingcraft-mc1.9-3.0.3.jar) 
    UCHIJAAAA   NashornLib{1.9.0-1.8.77-1.2.1} [Nashorn Library](NashornLib-1.9.0-1.8.77-1.2.1.5.jar) 
    UCHIJAAAA   craftingharmonics{1.9.0-1.3.12} [Crafting Harmonics](craftingharmonics-1.9.0-1.3.12.20.jar) 
    UCHIJAAAA   craftingtweaks{4.0.10} [Crafting Tweaks](craftingtweaks-mc1.9-4.0.10.jar) 
    UCHIJAAAA   Waila{1.7.0} [Waila](Waila-1.7.0-B2_1.9.jar) 
    UCHIJAAAA   DarkUtils{1.1.0.32} [Dark Utilities](DarkUtilities-1.9-1.1.0.33.jar) 
    UCHIJAAAA   DeathCounter{MC1.9.0v2} [DeathCounter](DeathCounter-MC1.9.0v2.jar) 
    UCHIJAAAA   ElecCore{1.4.270} [ElecCore](ElecCore-1.9-1.4.270.jar) 
    UCHIJAAAA   deepresonance{1.1.4beta30} [DeepResonance](deepresonance-1.9.0-1.1.4beta30.jar) 
    UCHIJAAAA   easyBreeding{4.0.1} [Easy Breeding](easybreeding-1.9-1.9-4.0.1.jar) 
    UCHIJAAAA   eggs{1.0} [Eat the Eggs!](Eat the Eggs-1.9-0.9 beta.jar) 
    UCHIJAAAA   emeraldobsidianmod{1.4} [Emerald & Obsidian tools/armor mod](emeraldobsidianmod-1.9-1.4%28beta%29.jar) 
    UCHIJAAAA   eplus{4.1.0.86} [Enchanting Plus](EnchantingPlus-1.9-4.1.0.97.jar) 
    UCHIJAAAA   endercompass{1.2.3} [Ender Compass](EnderCompass-1.9-1.2.3.jar) 
    UCHIJAAAA   endercrop{1.9-1.2-1836} [Ender Crop](endercrop-1.9.1.3.jar) 
    UCHIJAAAA   enderrift{0.16.3} [Ender-Rift](EnderRift-1.9-0.16.3.jar) 
    UCHIJAAAA   shetiphiancore{3.2.2} [ShetiPhian-Core](shetiphiancore-1.9.0-3.2.2.jar) 
    UCHIJAAAA   endertanks{1.3.3} [EnderTanks](endertanks-1.9.0-1.3.3.jar) 
    UCHIJAAAA   enderthing{0.6.6} [Enderthing](enderthing-1.9-0.6.6.jar) 
    UCHIJAAAA   enderutilities{0.5.3} [Ender Utilities](enderutilities-1.9-0.5.3.jar) 
    UCHIJAAAA   evilcraft{0.9.10} [EvilCraft](EvilCraft-1.9-0.9.10.jar) 
    UCHIJAAAA   ExtraBitManipulation{1.9-2.1.1} [Extra Bit Manipulation](ExtraBitManipulation-1.9-2.1.1.jar) 
    UCHIJAAAA   golems{4.03} [Extra Golems](ExtraGolems[1.9]-4.03.jar) 
    UCHIJAAAA   golems_waila{4.01} [Extra Golems Waila Addon](ExtraGolemsWailaAddon[1.9]-4.01.jar) 
    UCHIJAAAA   ezstorage{1.0} [EZ Storage](ezstorage-1.9-1.2.1.jar) 
    UCHIJAAAA   FFS{1.9-1.5.1.1.2} [Fancy Fluid Storage](Fancy Fluid Storage-1.9-1.5.1.1.2.jar) 
    UCHIJAAAA   fastleafdecay{1.6} [Fast Leaf Decay](FastLeafDecay-1.9-1.6.jar) 
    UCHIJAAAA   flatcoloredblocks{mc1.9-v2.2} [Flat Colored Blocks](flatcoloredblocks-mc1.9-v2.2.jar) 
    UCHIJAAAA   fluxducts{1.0.3} [Flux Ducts](fluxducts-mc1.9-1.0.3.jar) 
    UCHIJAAAA   samsfooddetails{1.0.1} [Food Details](FoodDetails-1.9-1.0.1.jar) 
    UCHIJAAAA   SilentLib{1.0.7} [Silent Lib](SilentLib-1.9-1.0.7-13.jar) 
    UCHIJAAAA   FunOres{1.2.4} [Fun Ores](FunOres-1.9-1.2.4-63.jar) 
    UCHIJAAAA   furnus{1.75} [Furnus](Furnus-1.9-1.75.jar) 
    UCHIJAAAA   pressure{1.3.1.13} [Pressure Pipes](pressure-1.3.1.13-mc1.9.jar) 
    UCHIJAAAA   advgenerators{0.9.20.8} [Advanced Generators](generators-0.9.20.8-mc1.9.jar) 
    UCHIJAAAA   glass_shards{1.6.1} [Glass Shards](glass_shards_mc1.9-1.6.1.jar) 
    UCHIJAAAA   graves{1.9-b10-universal} [Graves Mod](graves-1.9-b10-universal.jar) 
    UCHIJAAAA   Harvest{1.9-1.0.1-3} [Harvest](Harvest-1.9-1.0.1-3.jar) 
    UCHIJAAAA   HelpFixer{1.3.13} [HelpFixer](HelpFixer-1.9-1.3.13.jar) 
    UCHIJAAAA   hopperducts{1.4.7} [Hopper Ducts](hopperducts-mc1.9-1.4.7.jar) 
    UCHIJAAAA   ice_shards{1.4.1} [Ice Shards](ice_shards_mc1.9-1.4.1.jar) 
    UCHIJAAAA   IronChest{6.0.133.777} [Iron Chest](ironchest-1.9-6.0.133.777.jar) 
    UCHIJAAAA   journeymap{1.9-5.2.0b1} [JourneyMap](journeymap-1.9-5.2.0b1-unlimited.jar) 
    UCHIJAAAA   jet_packs{1.0.0} [Just JetPacks](JustJetpacks-1.0.0.jar) 
    UCHIJAAAA   lteleporters{1.1.1} [Lemon's Simple Teleporters](LemonsSimpleTeleporters-1.1.1.jar) 
    UCHIJAAAA   LunatriusCore{1.1.2.34} [LunatriusCore](LunatriusCore-1.9-1.1.2.34-universal.jar) 
    UCHIJAAAA   malisiscore{1.9-3.0.6} [MalisisCore](malisiscore-1.9-3.0.6.jar) 
    UCHIJAAAA   malisisswitches{1.9-2.0.0} [Malisis Switches](malisisswitches-1.9-2.0.0.jar) 
    UCHIJAAAA   am{1.0.0} [Marines Armor and Weapons](Marines Armor and Weapons 1.5.0.jar) 
    UCHIJAAAA   RadixCore{1.9-2.1.0} [RadixCore](RadixCore-1.9-2.1.0-universal.jar) 
    UCHIJAAAA   MCA{1.9-5.1.2} [Minecraft Comes Alive](MCA-1.9-5.1.2-universal.jar) 
    UCHIJAAAA   miab{1.4.2} [Meme in a bottle](MemeInABottle-1.4.2 1.9.jar) 
    UCHIJAAAA   testdummy{1.10} [MmmMmmMmmMmm](MmmMmmMmmMmm-1.10.jar) 
    UCHIJAAAA   modifiable_armor{2.0.1} [Modifiable Armor 2](Modifiable-Armor-1.0.1.jar) 
    UCHIJAAAA   morefood{1.15d} [More Foods](morefood-1.9-1854-1.15d.jar) 
    UCHIJAAAA   morematerials{1.0.20150301.0} [More Materials](morematerials-1.9.20160320.0.jar) 
    UCHIJAAAA   Morpheus{1.9-3.0.2} [Morpheus](Morpheus-1.9-3.0.2.jar) 
    UCHIJAAAA   AS_MultiMine{1.5.2} [Multi Mine](MultiMine-1.9.jar) 
    UCHIJAAAA   neotech{3.0.6} [NeoTech](NeoTech-1.9-3.0.6.jar) 
    UCHIJAAAA   netherportalfix{1.0} [Nether Portal Fix](netherportalfix-mc1.9-3.0.6.jar) 
    UCHIJAAAA   notenoughwands{1.3.3} [Not Enough Wands](notenoughwands-1.9.0-1.3.3.jar) 
    UCHIJAAAA   OAReborn{1.9-2.0.3.0} [OpenAutomation REBORN!](OAReborn-1.9-2.0.3.0.jar) 
    UCHIJAAAA   OALasers{1.0} [OALasers](OAReborn-1.9-2.0.3.0.jar) 
    UCHIJAAAA   samsocean{1.0.0} [Classic Oceans](OceanFloor-1.9-1.0.1.jar) 
    UCHIJAAAA   fodc{1.8.1} [Ore Dictionary Converter](OreDictionaryConverter-1.8.1.jar) 
    UCHIJAAAA   origin{9.0.3} [Origin](Origin-1.9-9.0.3.371.jar) 
    UCHIJAAAA   packingtape{0.5.1} [Packing Tape](Packing Tape-0.5.1.jar) 
    UCHIJAAAA   pedestal{1.9-0.1.5-alpha} [§cPedestal](pedestal-1.9-0.1.5-alpha.jar) 
    UCHIJAAAA   planttech{1.4.3} [Planttech](planttech-1.4.3 - 1.9.jar) 
    UCHIJAAAA   playerrugs{1.9-1.3.2} [Player Rugs](PlayerRugs-1.9-1.3.2.jar) 
    UCHIJAAAA   pocketnether{2.2} [Pocket Nether Link](PocketNetherLink-1.9-2.2.jar) 
    UCHIJAAAA   progressiveautomation{1.6.42} [Progressive Automation](ProgressiveAutomation-1.9-1.6.42.jar) 
    UCHIJAAAA   Psi{beta-31} [Psi](Psi-beta-31.jar) 
    UCHIJAAAA   reborncore{2.3.2.84} [RebornCore](RebornCore-1.9-2.3.2.84-universal.jar) 
    UCHIJAAAA   QuantumStorage{3.0.0} [QuantumStorage](QuantumStorage-1.9-3.0.0.jar) 
    UCHIJAAAA   Quark{beta-18} [Quark](Quark-beta-18.jar) 
    UCHIJAAAA   randomthings{3.7} [Random Things](RandomThings-MC1.9-3.7.jar) 
    UCHIJAAAA   rarmor{0.7.1} [Rarmor](Rarmor-0.7.1.jar) 
    UCHIJAAAA   RealisticTorches{1.5.4} [Realistic Torches](RealisticTorches-1.9-1.5.4.jar) 
    UCHIJAAAA   redstonepaste{1.7.3} [Redstone Paste](redstonepaste-mc1.9-1.7.3.jar) 
    UCHIJAAAA   xreliquary{1.9-1.3.1.342} [Reliquary](Reliquary-1.9-1.3.1.342.jar) 
    UCHIJAAAA   rftools{4.48} [RFTools](rftools-1.9.0-4.48.jar) 
    UCHIJAAAA   rftoolsdim{4.23beta38} [RFTools Dimensions](rftoolsdim-1.9.0-4.23beta38.jar) 
    UCHIJAAAA   Roguelike{1.5.0} [Roguelike Dungeons](RoguelikeDungeons-1.9-1.5.0.jar) 
    UCHIJAAAA   AS_Ruins{16.0} [Ruins Spawning System](Ruins-1.9.jar) 
    UCHIJAAAA   sciapi{1.3.0.3} [SciAPI](SciAPI-1.3.0.3.jar) 
    UCHIJAAAA   seasonsapi{1.1.3b} [Seasons API](SeasonsAPI-1.1.3b.jar) 
    UCHIJAAAA   sentek{1.2} [Sentek Runic Teleporters](Sentek-1.2.jar) 
    UCHIJAAAA   shadowmc{3.1.0} [ShadowMC](ShadowMC-1.9-3.2.1.jar) 
    UCHIJAAAA   simplegrinder{1.132.R19} [Simple Grinder](SimpleGrinder-1.132.R19.jar) 
    UCHIJAAAA   SimpleLabels{0.0.1} [SimpleLabels](simplelabels-0.0.2.jar) 
    UCHIJAAAA   su{1.0.11} [Simple Upgrades](simpleupgrades-1.0.11.jar) 
    UCHIJAAAA   SleepingBag{1.1.0} [Sleeping Bag](SleepingBag-1.9-1.1.0.jar) 
    UCHIJAAAA   SophisticatedWolves{3.6.1} [SophisticatedWolves](SophisticatedWolves-3.6.1.jar) 
    UCHIJAAAA   soulshardstow{1.9-2.3.9-25} [Soul Shards - The Old Ways](SoulShards-TOW-1.9-2.3.9-25.jar) 
    UCHIJAAAA   stellarsky{0.3.2.5b} [Stellar Sky](Stellar Sky-0.3.2.5b.jar) 
    UCHIJAAAA   StorageDrawers{1.9-3.1.6} [Storage Drawers](StorageDrawers-1.9-3.1.6.jar) 
    UCHIJAAAA   storagesilo{1.2.2} [StorageSilo](storagesilo-1.9-1.2.2.jar) 
    UCHIJAAAA   guru{1.3} [The Basic Elements](The Basic Elements 1.3.1.jar) 
    UCHIJAAAA   Trashcans{2.0.2} [Trashcans](Trashcans-1.9-2.0.2.jar) 
    UCHIJAAAA   uncraftingtable{1.0.4} [Uncrafting Grinder](UncraftingGrinder-1.9-1.0.4.jar) 
    UCHIJAAAA   underphangables{1.0.2} [Underp Hangables](underphangables-1.9-1.0.2.jar) 
    UCHIJAAAA   AS_UpdateCheck{1.1.7} [AtomicStryker Update Check Mod](UpdateCheckerMod-1.9.jar) 
    UCHIJAAAA   VeinMiner{0.31.5_build.forgePost18010} [Vein Miner](VeinMiner-1.9_0.31.5.forgePost18010.jar) 
    UCHIJAAAA   VeinMinerModSupport{0.31.5_build.forgePost18010} [Mod Support](VeinMiner-1.9_0.31.5.forgePost18010.jar) 
    UCHIJAAAA   vending{1.3.1} [vending](vending-1.9-1.3.4.jar) 
    UCHIJAAAA   WailaHarvestability{1.1.7} [Waila Harvestability](WailaHarvestability-mc1.9-1.1.7.jar) 
    UCHIJAAAA   ICSE{1.0.0.0} [I Can See Everything](Wawla-1.9-2.2.0.186.jar) 
    UCHIJAAAA   wawla{2.2.0.178} [What Are We Looking At](Wawla-1.9-2.2.0.186.jar) 
    UCHIJAAAA   weaponcaseloot{3.0.0-alpha} [WeaponCaseLoot](weaponcaseloot-3.0.0-alpha.jar) 
    UCHIJAAAA   dropchevsky{b13} [World Drop](WorldDrop_1.9_b13.jar) 
    UCHIJAAAA   YALSM{2.1.6} [Yet Another Leather Smelting Mod](yalsm-2.1.6-mc1.9.jar) 
    Loaded coremods (and transformers): 
LoadingPlugin (Quark-beta-18.jar)
  vazkii.quark.base.asm.ClassTransformer
NowWithRendering (redstonepaste-mc1.9-1.7.3.jar)
  net.fybertech.nwr.NowWithRendering
FldLoadingPlugin (FastLeafDecay-1.9-1.6.jar)
  com.olafski.fastleafdecay.FldTransformer
LoadingPlugin (RandomThings-MC1.9-3.7.jar)
  lumien.randomthings.asm.ClassTransformer
LoadingPluginBuildingBricks (BuildingBricks-1.9-2.0.1.jar)
  com.hea3ven.buildingbricks.core.load.ClassTransformerBuildingBricks
SeasonsPreLoader (SeasonsAPI-1.1.3b.jar)
  mod.mindcraft.seasons.asm.Transformer
MMFMLCorePlugin (MultiMine-1.9.jar)
  atomicstryker.multimine.common.fmlmagic.MMTransformer
MalisisCorePlugin (malisiscore-1.9-3.0.6.jar)
  net.malisis.core.util.chunkcollision.ChunkCollisionTransformer
  net.malisis.core.util.chunkblock.ChunkBlockTransformer
  net.malisis.core.renderer.transformer.MalisisRendererTransformer
  net.malisis.core.renderer.icon.asm.TextureMapTransformer
MalisisSwitchesPlugin (malisisswitches-1.9-2.0.0.jar)
  net.malisis.switches.asm.MalisisSwitchesTransformer
CoreMod (Aroma1997Core-1.9-1.0.5.0.b65.jar)

Reborn Core ASM (RebornCore-1.9-2.3.2.84-universal.jar)
  reborncore.asm.RebornClassTransformer
    Pulsar/tconstruct loaded Pulses: 
        - TinkerCommons (Enabled/Forced)
        - TinkerWorld (Enabled/Not Forced)
        - TinkerTools (Enabled/Not Forced)
        - TinkerSmeltery (Enabled/Not Forced)
        - TinkerGadgets (Enabled/Not Forced)
        - TinkerOredict (Enabled/Forced)
        - TinkerIntegration (Enabled/Forced)
        - TinkerFluids (Enabled/Forced)
        - TinkerMaterials (Enabled/Forced)
        - chiselsandbitsIntegration (Enabled/Not Forced)
        - craftingtweaksIntegration (Enabled/Not Forced)

```
List of loaded APIs: 
    * AbyssalCraftAPI (1.7.1) from AbyssalCraft-1.9-1.9.1.6.jar
    * actuallyadditionsapi (12) from ActuallyAdditions-1.9-r33.jar
    * AromaBackupAPI (1.0) from AromaBackup-1.9-1.0.1.0.b31.jar
    * Baubles|API (1.1.1.0) from Baubles-1.9-1.2.0.2.jar
    * BetterAchievements|API (0.2.5.19) from BetterAchievements-1.9-0.2.5.19.jar
    * BloodMagic|API (@VERSION@) from BloodMagic-1.9-2.0.0-38.jar
    * BotaniaAPI (78) from Botania-unofficial.r1.8-304.jar
    * CapabilityCore|base (1.01) from EnderRift-1.9-0.16.3.jar
    * CapabilityCore|energy (1.01) from EnderRift-1.9-0.16.3.jar
    * CapabilityCore|tweakable (1.01) from EnderRift-1.9-0.16.3.jar
    * ChiselAPI (0.0.1) from Chisel-MC1.9-0.0.4.14.jar
    * ChiselAPI|Carving (0.0.1) from Chisel-MC1.9-0.0.4.14.jar
    * ChiselsAndBitsAPI (1.8.1) from chiselsandbits-9.18.jar
    * CoFHAPI (1.8.9R1.2.0B1) from Bookshelf-API-1.9-3.0.4.jar
    * CoFHAPI|block (1.8.9R1.2.0B1) from ProgressiveAutomation-1.9-1.6.42.jar
    * CoFHAPI|energy (1.8.9R1.2.0B1) from ProgressiveAutomation-1.9-1.6.42.jar
    * CoFHAPI|tileentity (1.8.9R1.2.0B1) from ProgressiveAutomation-1.9-1.6.42.jar
    * compactstorage (v2.0.0a) from compactstorage-2.1-1.9-rc.jar
    * CraftingTweaks|API (4.1) from craftingtweaks-mc1.9-4.0.10.jar
    * DR-API (1.0.4-Beta) from deepresonance-1.9.0-1.1.4beta30.jar
    * ElecCore|Abilities (#API_VER#) from ElecCore-1.9-1.4.270.jar
    * FunOresAPI (3) from FunOres-1.9-1.2.4-63.jar
    * glass_shards|API (2.0) from glass_shards_mc1.9-1.6.1.jar
    * jeresources|API (0.4.5.27) from JustEnoughResources-1.9-0.4.5.27.jar
    * journeymap|client-api (1.2-SNAPSHOT) from journeymap-1.9-5.2.0b1-unlimited.jar
    * journeymap|client-api-display (1.2-SNAPSHOT) from journeymap-1.9-5.2.0b1-unlimited.jar
    * journeymap|client-api-event (1.2-SNAPSHOT) from journeymap-1.9-5.2.0b1-unlimited.jar
    * journeymap|client-api-model (1.2-SNAPSHOT) from journeymap-1.9-5.2.0b1-unlimited.jar
    * journeymap|client-api-util (1.2-SNAPSHOT) from journeymap-1.9-5.2.0b1-unlimited.jar
    * JustEnoughItemsAPI (3.1.0) from jei_1.9-3.3.3.200.jar
    * McJtyLib (1.9.0-1.8.4) from mcjtylib-1.9.0-1.8.4.jar
    * OAReborn|API (0.1) from OAReborn-1.9-2.0.3.0.jar
    * OAReborn|LasersAPI (0.1) from OAReborn-1.9-2.0.3.0.jar
    * origin|api (0.0.0) from Origin-1.9-9.0.3.371.jar
    * origin|foundation (0.0.0) from Origin-1.9-9.0.3.371.jar
    * pressureAPI (1.3.1.13) from pressure-1.3.1.13-mc1.9.jar
    * PsiAPI (2) from Psi-beta-31.jar
    * rarmorapi (3) from Rarmor-0.7.1.jar
    * reborncoreAPI (2.3.2.84) from RebornCore-1.9-2.3.2.84-universal.jar
    * SeasonAPI (1.0.0) from SeasonsAPI-1.1.3b.jar
    * SoulShardsTOW|API (0.1.5) from SoulShards-TOW-1.9-2.3.9-25.jar
    * stellarsky|API (2.2) from Stellar Sky-0.3.2.5b.jar
    * StorageDrawersAPI (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
    * StorageDrawersAPI|config (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
    * StorageDrawersAPI|event (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
    * StorageDrawersAPI|inventory (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
    * StorageDrawersAPI|pack (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
    * StorageDrawersAPI|registry (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
    * StorageDrawersAPI|render (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
    * StorageDrawersAPI|storage (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
    * StorageDrawersAPI|storage-attribute (1.7.10-1.2.0) from StorageDrawers-1.9-3.1.6.jar
    * VeinMinerApi (0.1) from VeinMiner-1.9_0.31.5.forgePost18010.jar
    * WailaAPI (1.3) from Waila-1.7.0-B2_1.9.jar
Profiler Position: N/A (disabled)
Player Count: 0 / 60; []
Is Modded: Definitely; Server brand changed to 'fml,forge'
Type: Dedicated Server (map_server.txt)`
```

**Comment by EdwinMindcraft** [comment]
Did it rain before ?

**Comment by EdwinMindcraft** [comment]
And if you can try to update your forge... It might be causing an issue...

**Comment by derSchotte** [comment]
Hi, i was on the edge of 2 biomes.. the one i was in it was snowing and the other had rain in it... so yes.

**Comment by derSchotte** [comment]
i had updated to the latest Version of Forge, but i try to find a nasty bug that crushes the client and server with an out of Synch Error... so i rollback to Forge 1900 with older Mods to find the Mod that causes the crash. You can test the pack if you want to... search on Curse for Twisted Twins Modpack.

**Comment by EdwinMindcraft** [comment]
Turns out the setRainfall method is client side only... I'll fix it when I can.

**Comment by MegaPunchQ** [comment]
I see that you released a new version for MC 1.9 with the winter season fix, could you also release one for MC 1.8.9?
